lib/compilers/glimmer/index.js

```diff
@@ -13,14 +13,39 @@ export default class GlimmerTemplatePrecompiler extends Filter {
 
   processString(content, relativePath) {
     let specifier = getTemplateSpecifier(this.rootName, relativePath);
-    return `export default ${this.precompile(content, { meta: { specifier, '<template-meta>': true } })};`;
+    let { contentWithReplacedPragmaTag, managerId } = extractPragmaTag(content);
+    let meta = { specifier, '<template-meta>': true };
+
+    if (managerId) {
+      meta.managerId = managerId;
+    }
+
+    let precompiledContent = this.precompile(contentWithReplacedPragmaTag, { meta });
+
+    return `export default ${precompiledContent};`;
   }
 
   precompile(content, options) {
     return precompile(content, options);
```

Comment:
it does, my initial thought was to implement AST transform but after speaking with @rwjblue we decided just to use regex since it's unclear which path is preferred: pragma vs. file path (this was faster to implement)